.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,6 +20,7 @@ https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-k
 > /kind failing-test
 > /kind feature
 > /kind flake
+> /kind regression
```

Comment:
/hold

As was discussed in the PR introducing this label, I think we'd probably consider regressions to also be bugs, and would want them labeled as such. Adding this label here alongside an "uncomment only one" instruction is confusing.

I'd also like to see more information provided when labeling issues as regressions, specifically:
* reproducer steps against a specific prior release that demonstrated different behavior

Reply:
Okay, how would we like to handle it in prow? Do we want to add `kind/bug` and `kind/regression` if the contributors specifies `/kind regression`?

Reply:
I would probably group the labels like this:

```
<!--
Add one of the following kinds:
/kind bug
/kind cleanup
/kind documentation
/kind feature

Optionally add one or more of the following kinds if applicable:
/kind api-change
/kind deprecation
/kind failing-test
/kind flake
/kind regression
-->
```

Reply:
Yes I can change that tomorrow. I think we have to preserve the `> ` before the kind otherwise prow would have issues in parsing the PR body.

I’ll check if I can enhance the kind parser for that as well.

Reply:
To apply @liggitt proposed layout we have to merge https://github.com/kubernetes/test-infra/pull/17261